internal/transform/scope-css.go

```diff
@@ -41,10 +41,15 @@ func ScopeStyle(styles []*astro.Node, opts TransformOptions) bool {
 		if n.FirstChild == nil || strings.TrimSpace(n.FirstChild.Data) == "" {
 			continue
 		}
+		scopeStrategy := 1
+		if opts.ScopedStyleStrategy == "class" {
+			scopeStrategy = 2
+		}
```

Comment:
Should these use an actual `type` instead of integers?

```go
// A NodeType is the type of a Node.
type ScopeStrategy uint32

const (
	ScopeStrategyWhere ScopeStrategy = iota
	ScopeStrategyClass
)
```